res/values-in/strings.xml

```diff
@@ -1115,7 +1115,7 @@
     <string name="setting_changed">Pengaturan diperbarui. Anda mungkin perlu menyambung kembali ke server agar dapat diterapkan.</string>
     <string name="Perform_10_1000_tricks_in_a_Trick_Mode_game_">Perform 10,000 tricks in a Trick Mode game.</string>
     <string name="Tricky">Penuh Akal</string>
-    <string name="SUPPORTER">VIP</string>
+    <string name="SUPPORTER">PENDUKUNG</string>
```

Comment:
I don't think this was supposed to be changed, since it's "VIP" in every other language, and it would be too long.

Reply:
Ok